nbs/nbdev.yml

```diff
@@ -3,7 +3,7 @@ project:
 
 website:
   title: "neuralforecast"
-  site-url: "https://Nixtla.github.io/neuralforecast/"
+  site-url: "https://nixtlaverse.nixtla.io/neuralforecast/"
```

Comment:
not sure why this changed when running `nbdev_export` but assuming its meant to be point to nixtlaversa?